x/checkpointing/keeper/bls_signer.go

```diff
@@ -40,7 +40,7 @@ func (k Keeper) SendBlsSig(ctx sdk.Context, epochNum uint64, lch types.LastCommi
 	}
 
 	// create MsgAddBlsSig message
-	msg := types.NewMsgAddBlsSig(epochNum, lch, blsSig, addr)
+	msg := types.NewMsgAddBlsSig(k.clientCtx.FromAddress, epochNum, lch, blsSig, addr)
```

Comment:
```suggestion
	msg := types.NewMsgAddBlsSig(k.clientCtx.GetFromAddress(), epochNum, lch, blsSig, addr)
```
no big difference though